README.md

```diff
@@ -15,15 +15,12 @@
 
 ## Setup and run
 
-0. Make sure you have Rust and Cargo installed with the `default` toolchain. With rustup,
-
-   - `curl https://sh.rustup.rs -sSf | sh -s`
-   - Or `Customise installation` then `Default toolchain?` set to `stable` and everything else empty.
-
-1. Run `git clone https://github.com/shramee/starklings-cairo1.git` to clone the repo
-2. Run `cd starklings-cairo1`
+1. Make sure you have Rust and Cargo installed with the `default` toolchain.  
+   With rustup `curl https://sh.rustup.rs -sSf | sh -s`
```

Comment:
This is technically not with rustup, but to install rustup

Reply:
AAh, woops 🤦

Reply:
Ohh, but it also runs rustup?